{{cookiecutter.project_shortname}}/README.md

```diff
@@ -0,0 +1,86 @@
+# {{cookiecutter.project_name}}
+
+This project was generated by the [dash-component-boilerplate](https://github.com/plotly/dash-component-boilerplate) it contains the minimal set of code required to create your own custom Dash component.
```

Comment:
I think we should move this to `CONTRIBUTING.md` and then have `README.md` contain something like:

```
# {{cookiecutter.project_name}}

{{cookiecutter.project_name}} is a Dash component library.

Get started with:
1. Install Dash and its dependencies: https://dash.plot.ly/installation
2. Run `python usage.py`
3. Visit http://localhost:8050 in your web browser

### Contributing

See [CONTRIBUTING.md](/CONTRIBUTING.md)
```